src/renderer/dx/DxRenderer.cpp

```diff
@@ -2003,7 +2003,7 @@ CATCH_RETURN();
     try
     {
         std::wstring fontName(desired.GetFaceName());
-        DWRITE_FONT_WEIGHT weight = DWRITE_FONT_WEIGHT_NORMAL;
+        DWRITE_FONT_WEIGHT weight = static_cast<DWRITE_FONT_WEIGHT>(desired.GetWeight());

```

Comment:
wow that was easy

Reply:
yeah uh spoiler, they're all just unsigned shorts under the hood. works for me.